source/index.txt

```diff
@@ -12,9 +12,118 @@ Connect your application to your database with one of our official libraries.
 
 The following libraries are officially supported by MongoDB. They are actively maintained, support new MongoDB features, and receive bug fixes, performance enhancements, and security patches.
 
-.. drivers-index-tiles::
+.. card-group::
+   :columns: 3
+   :layout: default
+   :type: drivers
+
+   .. card::
+      :headline: C
+      :url: /c/
+      :icon: /images/icons/c.svg
+      :icon-alt: C Driver icon
+
+   .. card::
+      :headline: C++
+      :url: /cxx/
+      :icon: /images/icons/c++.svg
+      :icon-alt: C plus plus Driver icon
+
+   .. card::
+      :headline: C#
+      :url: /csharp/
+      :icon: /images/icons/csharp.svg
+      :icon-alt: C sharp Driver icon
+
+   .. card::
+      :headline: Go
+      :url: https://www.mongodb.com/docs/drivers/go/current/
+      :icon: /images/icons/go.svg
+      :icon-alt: Go Driver icon
+
+   .. card::
+      :headline: Java
+      :url: /java-drivers/
+      :icon: /images/icons/java.svg
+      :icon-alt: Java Driver icon
+   
+   .. card::
+      :headline: Node.js
+      :url: https://www.mongodb.com/docs/drivers/node/current/
+      :icon: /images/icons/node.svg
+      :icon-alt: Node Driver icon
+
+   .. card::
+      :headline: PHP
+      :url: /php/
+      :icon: /images/icons/php.svg
+      :icon-alt: PHP Driver icon
+
+   .. card::
+      :headline: Python
+      :url: /python/
+      :icon: /images/icons/python.svg
+      :icon-alt: Python Driver icon
+   
+   .. card::
+      :headline: Ruby
+      :url: https://www.mongodb.com/docs/ruby-driver/current/
+      :icon: /images/icons/ruby.svg
+      :icon-alt: Ruby Driver icon
+
+   .. card::
+      :headline: Rust
+      :url: /rust/
+      :icon: /images/icons/rust.svg
+      :icon-alt: Rust Driver icon
+
+   .. card::
+      :headline: Scala
+      :url: /scala/
+      :icon: /images/icons/scala.svg
+      :icon-alt: Scala Driver icon
+
+   .. card::
+      :headline: Swift
+      :url: /swift/
+      :icon: /images/icons/swift.svg
+      :icon-alt: Swift Driver icon
+
+   .. card::
+      :headline: TypeScript
+      :url: https://www.mongodb.com/docs/drivers/node/current/fundamentals/typescript/
```

Comment:
Suggestion:

We have a new TypeScript landing page in the repo that you can link to:

```suggestion
      :url: /typescript/
```